src/user_input_compiler.rs

```diff
@@ -442,8 +442,8 @@ mod test {
   }
   
   #[test]
-  fn crowd() {
-    assert_eq!(Status, Input::from( format!("/crowd") ) )
+  fn crowd() { // cancelled with 0.3.0
+    assert_eq!(InvalidSyntax("Invalid CommandWord".into()), Input::from( format!("/crowd") ) )
   }
```

Comment:
I prefere it as a negative test, so it will not become implemented again by an accident.